must-gather/collection-scripts/builds.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+resource="builds"
+
+base_collection_path=$1
+namespace=$2
+if [ "${base_collection_path}" = "" ];then
+    echo "Base collection path for ${resource} is not passed. Exiting."
+    exit 0
```

Comment:
Here and elsewhere in the code, should we be reporting success status in the case that the scripts are not passed their required arguments? There may well be a reason we need to do so, but if we can it would likely be better to exit with an error status to avoid silent failures.